### DISCUSSION TITLE
'Your PC Ran Into An Issue' Error

### DISCUSSION THREAD

**Discussion by kwaduo** [utmapp/UTM#3065] [UTM Help (macOS)]
I have followed CONATH's tutorial and have got the VM running up to the point that i load on the ISO. At that point, after restarting I get a Your PC Ran into an Issue error and force restarts. This then breaks the VM and I have to delete it and start over.

https://user-images.githubusercontent.com/89603871/131089122-c17f11e4-da94-4281-aee7-e990f51e3ba7.mov

**Comment by conath** [comment]
What is your UTM version, macOS version and device type (e. g. Mac M1 or Mac Intel) ? How much Memory does your Mac have?
Thanks
